buildroot/share/unit-tests/TestVector.hpp

```diff
@@ -74,58 +74,58 @@
 #endif
 
 /** All testing code frameword is declared inside this namespace */
-namespace Test
-{
+namespace Test {
+
   inline bool isUnderDebugger() {
-#ifdef _WIN32
-    return (IsDebuggerPresent() == TRUE);
-#elif defined(__linux__)
-    static signed char testResult = 0;
-    if (testResult == 0) {
-      testResult = (char) ptrace (PT_TRACE_ME, 0, 0, 0);
-      if (testResult >= 0) {
-        ptrace (PT_DETACH, 0, (caddr_t) 1, 0);
-        testResult = 1;
+    #ifdef _WIN32
+      return (IsDebuggerPresent() == TRUE);
+    #elif defined(__linux__)
+      static signed char testResult = 0;
+      if (testResult == 0) {
+        testResult = (char) ptrace (PT_TRACE_ME, 0, 0, 0);
+        if (testResult >= 0) {
+          ptrace (PT_DETACH, 0, (caddr_t) 1, 0);
+          testResult = 1;
+        }
```

Comment:
I'll try to report those change in the other PR then (or I can close the other PR either way) #21494